squid-blockexplorer/src/blocks/processCalls.ts

```diff
@@ -0,0 +1,51 @@
+import { CallItem, Context } from '../processor';
+import { Block, Account } from '../model';
+import { createExtrinsic, createCall } from './utils';
+import { ExtrinsicsMap, CallsMap } from './types';
+
```

Comment:
superrr nitpick but we've done `no-non-null-assertion` a few times now, maybe just disable it for the file?

I will say I do like being very specific about it.

```suggestion
/* es-lint-disable @typescript-eslint/no-non-null-assertion */
```

Reply:
If it wasn't `no-non-null-assertion` I would agree, but since this can break the app I would prefer to be very specific